content/people/tim-au-yeung.yaml

```
@@ -1,5 +1,6 @@
 name: Tim Au Yeung
-type: phd
+type: alumni
+past: phd
```

Reverting on suggestion by @Ehud-Sharlin
```suggestion
type: phd
```